### PR TITLE
Create a sample to show how to tag VMs as they are created.

### PR DESCRIPTION
**Pull request by mattmoor**

### CODE REVIEW

File: cmd/sample/tag-created-vms/README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+Now let's put it all together:
+```go
+package main
+
+import (
+	"context"
+	"log"
+
+	cloudevents "github.com/cloudevents/sdk-go/v1"
```
Comment (n3wscott) [inline cmd/sample/tag-created-vms/README.md]: v1 -> v2

**Review by n3wscott — COMMENTED** [review]
I think the only thing you need to do is change v1 to v2

**Comment by n3wscott** [comment]
In the next release of cloudevents preview, this changes, we are going to break the contract on client, it will return a Result rather than an error, and you can check for ACK: https://github.com/cloudevents/sdk-go/pull/425

File: cmd/sample/tag-created-vms/README.md (hunk 1 above)
Comment (n3wscott) [reply cmd/sample/tag-created-vms/README.md]: still v1

Comment (mattmoor) [reply cmd/sample/tag-created-vms/README.md]: I knew I'd regret inlining that file, that didn't take long.

**Comment by vaikas** [comment]
I'd rather have the samples in samples/* directory.

File: cmd/sample/tag-created-vms/README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+# Tag Created VMs
```
Comment (vaikas) [inline cmd/sample/tag-created-vms/README.md]: let's move this to:
samples/tag-created-vms/README.md

Comment (mattmoor) [reply cmd/sample/tag-created-vms/README.md]: done

**Pull request merged**